### ISSUE TITLE
Client crash

### ISSUE THREAD

**Issue by Enjoyop** [CrazorTheCat/BSDS-V43#2]
every time i don't start a war the game crashes for all sorts of reasons.
-At least the game modes are decent.
I guess this is not Arxan frida protection.
I manipulate the client without frida but it still crashes.
I think there is an error in the packages. sourced from accessories and star powers.
What do you think is the root cause of this problem?

**Comment by CrazorTheCat** [comment]
yeah i saw some crashes happening in menu when clicking some buttons randomly but honestly i have no idea where it come from i should do further inspection to find the problem

**Comment by breaddev228** [comment]
дебил потому что
